### PR TITLE
Add calendar script

### PR DESCRIPTION
**Pull request by mbpope**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,47 +1,161 @@
-<!DOCTYPE html>
```
Comment (theryangeary) [inline index.html]: Please convert to UNIX style line endings like so: https://stackoverflow.com/questions/16768776/convert-line-endings

This way git won't think the entire file changed

File: style.css [hunk 2]
```diff
@@ -1,57 +1,64 @@
...
+.calendar small {
+  font-size: 50%;
+}
+
+.calendar p {
+  margin-bottom: 0;
+}
+
+.event {
+    display: block;
```
Comment (theryangeary) [inline style.css]: Please use two spaces, not four, to match the formatting of the rest of the code.

**Review by theryangeary — CHANGES_REQUESTED** [review]
I'm going to need to see documentation about what config.js should be like, and we're going to need a plan to provide that information to GH Pages before this can go through. Great work so far!

File: index.html [hunk 3]
```diff
@@ -1,47 +1,161 @@
...
+    <meta charset="UTF-8">
+    <title>Linux Users Group @ UD</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Staatliches" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+  </head>
+
+  <body>
+      <script type='text/javascript' src='config.js'></script>
+      <script>
```
Comment (theryangeary) [inline index.html]: It would also be good to move this into a separate .js file

**Comment by mbpope** [comment]
> We're going to need a plan to provide that information to GH Pages before this can go through.

Actually all of the API information will be visible on the client side anyway. The protection is on Google's end (we have to configure OAuth to only allow requests from udellug.github.io). The only reason I excluded my API credentials from the commit were because they are configured to allow requests from http://localhost and I don't want that leaking. But in the final production we don't need to hide anything. In the next commit I'll remove `config.js` from `.gitignore` and commit a clean version without my development credentials.

File: index.html (hunk 1 above)
Comment (mbpope) [reply index.html]: I did this, although because I changed the line endings, once again it thinks I changed the whole file 🤦‍♂

**Comment by theryangeary** [comment]
I am applying these changes manually since there were some conflicts.

**Pull request closed without merging**